### PR TITLE
Payment terms migration

### PR DESCRIPTION
**Pull request by mehalshah**
Create db objects for payment terms

### CODE REVIEW

File: dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreatePdPaymentTerms < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_payment_terms do |t|
+      t.references :regional_partner, foreign_key: true
+      t.date :start_date
+      t.date :end_date
+      t.string :course
+      t.string :subject
+      t.text :properties
```
Comment (ashercodeorg) [inline dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb]: Do any of these have defaults? Should any of these be non-NULL?

File: dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreatePdPaymentTerms < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_payment_terms do |t|
```
Comment (ashercodeorg) [inline dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb:3]: Are any indexes (beyond `regional_partner_id`) necessary?

File: dashboard/db/schema.rb [hunk 3]
```diff
@@ -11,7 +11,6 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema.define(version: 20170407201709) do
```
Comment (ashercodeorg) [inline dashboard/db/schema.rb]: Should the timestamp (`version`) be updated?

File: dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb (hunk 2 above)
Comment (mehalshah) [reply dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb:3]: No. We'll only be selecting on regional ID.

File: dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb (hunk 1 above)
Comment (mehalshah) [reply dashboard/db/migrate/20170406215340_create_pd_payment_terms.rb]: There's no default that makes sense for any of these. Properties should be non-Null, the rest can be null. I'll make the change.

File: dashboard/db/schema.rb (hunk 3 above)
Comment (mehalshah) [reply dashboard/db/schema.rb]: Thought I did - will try again

**Pull request merged**